### PR TITLE
Issue 423

### PR DESCRIPTION
**Pull request by sabbelasichon**

### CODE REVIEW

File: src/Rector/v8/v0/TimeTrackerInsteadOfNullTimeTrackerRector.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, GeneralUtility::class)) {
+            return false;
+        }
+
+        return $this->isName($node->name, 'makeInstance');
+    }
+
+    private function isObjectMangerCall(Node $node): bool
```
Comment (ayacoo) [inline src/Rector/v8/v0/TimeTrackerInsteadOfNullTimeTrackerRector.php:142]: Manager

File: captainhook.json [hunk 2]
```diff
@@ -16,17 +16,17 @@
                 "conditions": []
             },
             {
-                "action": "vendor/bin/ecs check --ansi",
+                "action": "composer run check-style",
```
Comment (ayacoo) [inline captainhook.json:19]: Do the composer changes need to be made in this commit? So you have a dependency on the rector? Otherwise it would be a separate commit for me.

Comment (sabbelasichon) [reply captainhook.json:19]: It is a separate commit.
